Model/Api/DiscountCodeValidation.php

```diff
@@ -118,7 +118,7 @@ private function handleRequest()
         if (isset($requestArray['data'])) {
             $requestArray = $requestArray['data'];
         }
-        
```

Comment:
I don't mind formatting changes, in fact I think it's good, but next time could you put those in a separate commit to make reviewing easier?